### ISSUE TITLE
Tooltips disappear when they are hovered over

### ISSUE THREAD

**Issue by clane** [vmware/clarity#3880]
## Describe the bug

Tooltips disappear when they are hovered over

## How to reproduce

- Go to http://cx-guidelines.vmware.com/clarity-test-app2
- Activate the "Tooltip Demos" links
- Hover over the tooltip icon under the "Angular Component" heading and observe that the text "Lorem" appears in a speech bubble
- Hover over the speech bubble and observe that it disappears. It should remain visible when hovered over. 
- Go to https://clarity.design/documentation/tooltips and repeat the steps above with the HTML (non Angular) tooltips

## Expected behavior

Tooltips should not disappear when they are hovered over.  If a pointer hover can trigger the appearance of additional content, then it should be possible to move the pointer over the additional content without the additional content disappearing.  One benefit of following this criteria is that low vision users using zoom functionality will be able to move the zoomed viewport over the newly displayed content without it disappearing.

WCAG 2.1 Guideline: 
1.4.13 Content on Hover or Focus
(Level AA)
https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus

Recommendation:
Follow the example at http://whatsock.com/tsg/Coding%20Arena/Tooltips/Tooltip%20(Internal%20Content)/demo.htm#. 

## Versions

**App**

* Angular:  8
* Node:  NA
* Clarity:  v2.0.0-rc.1

**Device:**

* Type:  mouse
* OS: Windows, Mac
* Browser: any
* Version:any

**Comment by gnomeontherun** [comment]
I think this might be addressed by using new popovers, but flagging this as a bug to track it.

**Comment by AlexMarcuBytex** [comment]
I ll investigate this issue. You can assign me. Thanks!!

**Comment by gnomeontherun** [comment]
This is one we need to consider more carefully as I look at it, because CSS only version isn't possible to fix this problem (`:hover` properties are used to display the tooltip, but moving the mouse will always hide the content). 

For the Angular component we could address this somehow, but this is first blocked by moving tooltips to new popovers. We'd have to implement some kind of mouse tracking and that depends on using new popovers first.

**Comment by chlane** [comment]
Unfortunately there is no authoritative tooltip example from the ARIA working group at this time.  The design pattern is work in progress that does not yet have task force consensus. Progress and discussions are captured in issue 128. Work to develop a tooltip example is tracked by issue 127.
This http://whatsock.com/tsg/Coding%20Arena/Tooltips/Tooltip%20(Internal%20Content)/demo.htm# appears to be accessible, allowing the tooltip to be hovered over and not disappear.
